docs/pages/eas/workflows/syntax.mdx

```diff
@@ -303,30 +396,120 @@ jobs:
     # @end #
     steps:
       - id: step_1
+        # @info #
         run: set-output test "hello world"
-
+        # @end #
   job_2:
     needs: [job_1]
     steps:
       # @info #
-      - run: echo ${{ needs.job_1.outputs.output_1 }}
+      - id: step_2
+        run: echo ${{ needs.job_1.outputs.output_1 }}
       # @end #
 ```
 
-### Built-in EAS functions
+### `jobs.<job_id>.steps`
+
+A job contains a sequence of tasks called `steps`. Steps can run commands or use an action.
```

Comment:
I don't think we've ever mentioned functions as "actions". Doesn't this resemble GitHub Actions too much (to the point where people may think they may use `uses: jonsamp/setup-node@v4` which is not the case?